docs/user-guide/work-packages/edit-work-package/README.md

```diff
@@ -94,6 +95,12 @@ You can copy a direct link to a specific work package comment in OpenProject. To
 
 ![Copy a link to a work package comment in OpenProject](openproject_user_guide_wp_edit_comment_copy_link.png)
 
+### Internal comments (Enterprise add-on)
+> [!NOTE]
+> Internal comments are an Enterprise add-on and can only be used with [Enterprise cloud](../../../enterprise-guide/enterprise-cloud-guide/) or [Enterprise on-premises](../../../enterprise-guide/enterprise-on-premises-guide/). An upgrade from the free Community edition is easily possible.
+
+It is possible to leave comments that are only visible to a certain group of people. These are referred to as internal comments. 
```

Comment:
```suggestion
It is possible to add comments that are only visible to a certain group of people. These are referred to as internal comments. 
```
cc @psatyal

Reply:
@MayaBerd - I tagged @psatyal in case we want to have additional information on "visible to a certain group of people"- it might raise questions as is.